aggregate_spatial.json

```diff
@@ -0,0 +1,80 @@
+    "categories": [
+        "cubes",
+        "aggregate & resample"
+    ],
+    "parameters": [
+        {
+            "name": "data",
+            "description": "A raster data cube.",
+            "schema": {
+                "type": "object",
+                "subtype": "raster-cube"
+            }
+        },
+        {
+            "name": "geometries",
+            "description": "Geometries as GeoJSON on which the aggregation will be based.",
+            "schema": {
+                "type": "object",
+                "subtype": "geojson"
+            }
+        },
+        {
+            "name": "reducer",
+            "description": "A reducer to be applied on all values of each geometry. A reducer is a single process such as ``mean()`` or a set of processes, which computes a single value for a list of values, see the category 'reducer' for such processes.",
+            "schema": {
+                "type": "object",
+                "subtype": "process-graph",
+                "parameters": [
+                    {
+                        "name": "data",
+                        "description": "An array with elements of any type.",
+                        "schema": {
+                            "type": "array",
+                            "items": {
+                                "description": "Any data type."
+                            }
+                        }
+                    }
+                ]
+            }
+        },
+        {
+            "name": "target_dimension",
+            "description": "The new dimension name to be used for storing the results. Defaults to `result`.",
+            "schema": {
+                "type": "string"
+            },
+            "default": "result",
+            "optional": true
+        }
+    ],
+    "returns": {
+        "description": "A vector data cube with the computed results. The vector data cube implicitly gets restricted to the bounds of the geometries as if ``filter_spatial()`` would have been used with the same values for the corresponding parameters immediately before this process.\n\nThe computed value is stored in dimension with the name that was specified in the parameter `target_dimension`.\n\nThe computation also stores information about the total count of pixels (valid + invalid pixels) and the number of valid pixels (see ``is_valid()``) for each geometry. These values are stored as new dimension with a dimension name derived from `target_dimension` by adding the suffix `_meta`. The new dimension has the dimension labels `total_count` and `valid_count`.",
```

Comment:
Why do we need to mention anything about how "The vector data cube implicitly gets restricted to the bounds of the geometries"? The vector data cube doesn't change at all, aside from gaining an extra attribute/dimension that holds the aggregation result.

Reply:
This was requested by some partners in #101.

Reply:
Huh, but that one is talking about the raster data cube, not the vector one...

Reply:
Your input is a raster data cube, which the vector data cube will be built upon. Thus the vector cube inherits that restriction.

Reply:
What do you mean? The input is a vector geometry, and the process simply adds a dimension to the vector geometry. The raster input is just there as the source of the dimension values, it is read-only.

#101 was actually talking about how the raster data cube gets cropped to the polygons as an optimisation, so that all the rest of the code doesn't need to run on areas that will not be used in the end. But that's already implied (backends should not do work that is not needed) and backend-specific. So I don't see any reason to mention it in the description.

Reply:
I moved things around a bit in commit https://github.com/Open-EO/openeo-processes/pull/131/commits/cde82763da87d9b7779aadeef79fe56a18a9d6a7, which indeed seems to make a bit more sense. Do you agree, @jdries?